### PR TITLE
docs: add JupyterLab integration docs

### PR DESCRIPTION
**Pull request by renan-r-santos**
What do you guys think about the Binder link? My idea was to help people who want to do a quick test of using Pixi + JupyterLab before adopting it.

### CODE REVIEW

**Comment by dhirschfeld** [comment]
I always preferred manually specifying my kernels with `conda/mamba` rather than using `nb_conda_kernels`. It should be possible to do the same with `pixi` without any magic (though some people do prefer the magic!).

Manually creating a `jupyter kernelspec` like the below should do the job for people who like doing it the hard way! 😅 
```json
{
  "argv": [
    "pixi", "run", "--manifest-path", "~/pixi/envs/dev312",
    "python", "-m", "ipykernel_launcher",
    "-f", "{connection_file}"
  ],
  "display_name": "dev312",
  "language": "python",
  "env": {
    "A_VARIABLE": "true",
  },
  "metadata": {
    "debugger": true
  }
}
```

File: docs/ide_integration/jupyterlab.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+Having installed the dependencies, create a folder for your notebooks and start JupyterLab using the following command:
+
+```bash
+mkdir -p pixi-notebooks
+pixi run jupyter lab --notebook-dir=pixi-notebooks
+```
+
+This will start JupyterLab and open it in your browser.
+
+![JupyterLab launcher screen showing Pixi Kernel](https://raw.githubusercontent.com/renan-r-santos/pixi-kernel/main/assets/launch-light.png#only-light)
```
Comment (ruben-arts) [inline docs/ide_integration/jupyterlab.md:33]: Nice that you already add this!

**Review by ruben-arts — APPROVED** [review]

**Comment by renan-r-santos** [comment]
I'm converting this to a draft for now. I released a new version of `pixi-kernel` and now it is capable of supporting in theory any other Jupyter kernels. It already supports Python, R, Bash and C++.
I need to update this PR to reflect that and also address some comments from Wolf. I'll let you guys know when this is ready for review again. Tks.

**Comment by renan-r-santos** [comment]
@ruben-arts @wolfv I believe this is ready for another pass.
`pixi-kernel` 0.2.0 added support for additional kernels, such as R, C++ and Bash. I updated the `pixi-kernel` section accordingly and added a few words on when you might want to use it.

**Comment by wolfv** [comment]
Thanks @renan-r-santos!

**Pull request merged**